tests/jquery.flot.touchNavigate.Test.js

```diff
@@ -315,6 +358,108 @@ describe("flot touch navigate plugin", function () {
 
       });
 
+      it('should pan the plot for two fingers with constant distance',function() {
```

Comment:
We need to clean up things here. The touchNavigate plugin deals with high level messages like: pan, pinch and double tap. The describe sections of this file should be named accordingly.

This test is testing panning but as a corner case for pinch, so it should be moved in the pinch section.

There should be an almost identical test which makes sure no zoom is happening even when there is a small variation in distance.